### DISCUSSION TITLE
dp train problem

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#471] [General]
Now i have three type train sets BAs, GaN and BAs-GaN,respectively. But the type_map of data file are (B,As), (Ga,N) and (As,B,N,Ga), so how can i define rightly "type_map" format in input.json file. Any discussion is welcomed.

**Comment by njzjz** [comment]
You need to use a same type map, even if a type is missing in some system.

**Reply by WU2JING** [reply]
Ok，thanks.
